internal/services/compute/linux_virtual_machine_resource.go

```diff
@@ -1326,6 +1326,17 @@ func resourceLinuxVirtualMachineUpdate(d *pluginsdk.ResourceData, meta interface
 			case "stopped":
 				// VM already stopped, no shutdown needed anymore
 				shouldShutDown = false
+			case "starting":
+				// send a duplicate start command to ensure Virtual Machine is not in transitioning state which doesn't accept further command
+				log.Printf("[DEBUG] Starting %q", id)
+				future, err := client.Start(ctx, id.ResourceGroup, id.Name)
```

Comment:
1. Will the backend directly return an error if it receives a duplicated command?
2. Since `instanceView.Statuses` is an array, will it contain `starting` and `deallocating` simultaneously?

Reply:
I'd agree with @ms-zhenhua here, from memory the Virtual Machine Start API errors if you try and call start multiple times - instead of doing this, we could introduce a `WaitForState` func which checked the `status` of the Virtual Machine?